tests/observability/metrics/conftest.py

```diff
@@ -997,11 +997,13 @@ def vnic_info_from_vm_or_vmi(request, running_metric_vm):
     vm_instance = (
         running_metric_vm.vmi.instance.spec if request.param == "vmi" else running_metric_vm.instance.spec.template.spec
     )
```

Comment:
```suggestion
    vm_interface = (
        running_metric_vm.vmi.interfaces if request.param == "vmi" else running_metric_vm.get_interfaces()
    )
```

How about using the property/function?

Reply:
The vm instance is used for `vnic_name` and `model` , so in order to avoid duplicate code/extra api call I did the vm_instance and use it to get the fields with one single api call.